overrides/fips.yaml

```
@@ -4,6 +4,8 @@ k8s_image_registry: docker.io/mesosphere
 fips:
   enabled: true
```

Or something like that
```suggestion
fips:
  enabled: true
  # when set to true and FIPS mode is not already enabled, FIPS mode will be enabled and the machine will be rebooted
  # only works on RHEL 8 
  configure: false
```

---

yes actually i love this. 

we should guard all the tasks in `fips` role  to only run if `configure == true`

---

Looks like you forgot to make this change too.
You will also need to override this configuration in KIB's CI.

---

right now im keeping this bit locally and for CI I'm planning on doing it in an extra override passed  to here.

---

added these to CI. and made the default false